.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-.env.*
\ No newline at end of file
+.env.*
```

Comment:
.env.* is covering all files starting with .env. , so not necessary to include .env.test or .env.development in the .gitignore file